Error: In order to serve as an `id`-reference for `aria-labelledby`, the `label` must have an `id` attribute, not `for`. The `for` attribute is itself an `id`-reference to an  associated form control, which is not applicable in this case.
Error: For broadest accessibility, implicitly associated `label` elements should also be explicitly associated with their control, using `for`.
Error: Checkboxes in a group should use a `name` value that ends with "[]" to allow server-side code to collect the values as an array (or similar). Note: The `div` wrapper is unnecessary.
diff --git a/output/Checkbox Group with Accessible Description/4/c0c7caf0-f4db-488b-be25-beb53f159868.html b/output/Checkbox Group with Accessible Description/4/c0c7caf0-f4db-488b-be25-beb53f159868.html
index e2bf700..7e7ec36 100644
--- a/output/Checkbox Group with Accessible Description/4/c0c7caf0-f4db-488b-be25-beb53f159868.html	
+++ b/output/Checkbox Group with Accessible Description/4/c0c7caf0-f4db-488b-be25-beb53f159868.html	
@@ -1,7 +1,16 @@
-<label for="fruits">What fruits do you like?</label>
+<label id="fruits">What fruits do you like?</label>
 <p id="fruits-description">You can change this later</p>
 <div role="group" aria-labelledby="fruits" aria-describedby="fruits-description">
-  <label><input type="checkbox" name="fruits" value="bananas"> Bananas</label>
-  <label><input type="checkbox" name="fruits" value="oranges"> Oranges</label>
-  <label><input type="checkbox" name="fruits" value="apples"> Apples</label>
+  <label for="bananas">
+    <input type="checkbox" name="fruits[]" id="bananas" value="bananas">
+    Bananas
+  </label>
+  <label for="oranges">
+    <input type="checkbox" name="fruits[]" id="oranges" value="oranges">
+    Oranges
+  </label>
+  <label for="apples">
+    <input type="checkbox" name="fruits[]" id="apples" value="apples">
+    Apples
+  </label>
 </div>
\ No newline at end of file
